app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -46,6 +46,11 @@ def start
     image = target_entity
     return queue_signal(:abort_job, "no image found", "error") unless image
     return queue_signal(:abort_job, "cannot analyze non docker images", "error") unless image.docker_id
+    return queue_signal(
+      :abort_job,
+      "smartproxy role missing",
+      "error"
+    ) unless MiqServer.my_server.has_active_role?("smartproxy")
```

Comment:
I don't understand @moolitayer.  We can route the work to a server with smartproxy enabled by using the role field in the miqqueue row.  I'm not sure how this work is being picked up by the wrong server.